src/components/Carousel/Carousel.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+  <section class="carousel">
+    <div class="carousel__container">
+      <button
+        id="left-btn"
+        class="carousel-control carousel-control--left"
+        @click="handlePrevious"
+      >
+        <img
+          src="../../assets/icons/left.svg"
+          alt="left arrow"
+        >
+      </button>
+      <div
+        id="carousel-images"
+        class="carousel-images"
+      >
+        <CarouselItem
+          v-for="(photo, index) in photos"
+          :key="photo.id"
+          :active="activeIndex"
+          :index="index"
+          :photo="photo"
```

Comment:
Same naming idea applies here as you are passing down not a photo/image, but rather an object with shop item data. 🤔